src/containers/forms/OrganizationForm/index.tsx

```diff
@@ -0,0 +1,197 @@
+import { ErrorMessage, Field, Form, Formik } from 'formik';
+import moment from 'moment';
+import React, { useState } from 'react';
+import { Redirect } from 'react-router';
+import { Button, Label } from 'reactstrap';
+import { FormGroup } from 'reactstrap';
+import * as Yup from 'yup';
+import {
+  ACTIVE,
+  NAME,
+  NO,
+  OPENSRP_ORGANIZATION_ENDPOINT,
+  ORGANIZATION_LABEL,
+  ORGANIZATIONS_LIST_URL,
+  REQUIRED,
+  SAVE,
+  SAVING,
+  YES,
+} from '../../../constants';
+import { generateNameSpacedUUID } from '../../../helpers/utils';
+import { OpenSRPService } from '../../../services/opensrp';
+
+const OrgFormNameSpace = '9a4c8cb0-df70-11e9-b38b-57f114a50538';
+
+/** default type value for organizations */
+const defaultOrganizationType = {
+  coding: [
+    {
+      code: 'team',
+      display: 'Team',
+      system: 'http://terminology.hl7.org/CodeSystem/organization-type',
+    },
+  ],
+};
+
+/** yup validation schema for teams Form input */
+export const OrgSchema = Yup.object().shape({
+  active: Yup.boolean(),
+  identifier: Yup.string(),
+  name: Yup.string().required(REQUIRED),
+});
+
+/** interface for data fields for team's form */
+export interface OrganizationFormFields {
+  active: boolean;
+  identifier: string;
+  name: string;
+  type: typeof defaultOrganizationType;
```

Comment:
@p-netm can we create an issue to create an interface for this?  It is not usual, and imho not recommended, to use `typeof` with an object that you already have, and thus should be able to create an interface for.  it also sets us up for more DRY code in the future when we have other objects of this type/shape.